source/docs/api-reference/index.rst

```diff
@@ -3,6 +3,29 @@
 API Reference
 =============
 
+This section highlights the functionality that is exposed as part of the Phoenix Pro API. This includes the base API, simulation and detailed example projects.
+
+.. grid:: 1 2 3 3
+   :gutter: 3
+
+   .. grid-item-card:: API Usage
+      :link-type: doc
+      :link: api-usage/index
```

Comment:
There are a couple of other places where we have these grid cards, such as the API Usage index.rst. Could you add links to those cards as well?

Reply:
Could you clarify what you mean by **those cards**? Specifically, what is **those** referring to?

Reply:
Looking over everything again, it looks like it's just the three cards on the API Usage index.rst page. I feel like all three cards (Configs, Control Requests, and Signals) should probably link to their respective pages.

Reply:
Got it. Thanks.